recipes/psyinf-gmtl/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(psyinf-gmtl REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} ../test_package/test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE psyinf-gmtl::psyinf-gmtl)
```

Comment:
names in CMakeDeps & cmake_find_package are not consistent

Reply:
I just tried to use the recipe with cmake multi environment. Do we need to set 
 self.cpp_info.names["cmake_find_package"] = "gmtl"
 self.cpp_info.names["cmake_find_package_multi"] = "gmtl"
?